### PR TITLE
feat: use new invitation spec

### PR DESCRIPTION
**Pull request by samsiegart**
fixes: https://github.com/Agoric/dapp-inter/issues/68

Tested manually opening, adjusting, and closing a vault.

Blocked: requires https://github.com/Agoric/wallet-app/pull/59 and https://github.com/Agoric/agoric-sdk/pull/6996

### CODE REVIEW

File: src/service/vaults.ts
Comment (turadg) [inline src/service/vaults.ts:338]: consider annotating the type here to statically validate the shape

File: src/service/vaults.ts
Comment (turadg) [inline src/service/vaults.ts:406]: why does this need to be serialized now?

**Review by turadg — COMMENTED** [review]
Love to see all the red!

I had a question before approval.

Also there's a lint warning on vaults.ts for an unused type. I think we should make that be an error but not a blocker.

File: src/service/vaults.ts
Comment (samsiegart) [reply src/service/vaults.ts:406]: The wallet app now expects the invitation spec to be capdata. This supports the `agoricContract` proposal which has a `Brand` as an arg. It was easier this way than making the wallet-app check if it's capdata or not.

Comment (turadg) [reply src/service/vaults.ts:406]: ok, makes sense that what goes between the apps is marshaled.

**Review by turadg — APPROVED** [review]

File: package.json
Comment (turadg) [inline package.json:11]: We don't have a separate script for this in other packages. It's a sort of lint so we have "lint" do it. Consider aligning with other repos.

File: package.json
Comment (turadg) [inline package.json:12]: Format should only affect white space and eslint fixers do more than that. They can break code. Please remove eslint from format.

Also, why only src? A bunch of non-src files were reformatted in this PR. Please omit those changes or make this continue to reformat them.

**Review by turadg — APPROVED** [review]
Some requests that I trust you'll do before merge. Approving now given dependencies so you don't need another review to land.

File: package.json
Comment (samsiegart) [reply package.json]: Done

File: package.json
Comment (samsiegart) [reply package.json]: Done

**Pull request merged**
